index.html

```diff
@@ -460,6 +457,9 @@
             </ol>
           </li>
           <li>Let <var>serializedModifierData</var> be an empty list.
+          <li>If <var>details.paymentRequestId</var> was not provided during
+          construction, populate the field with an [[RFC4122]] UUID.
```

Comment:
Oh, I see. I don't think we have consensus to make this a hard requirement. Also, avoid modifying the incoming data, create a new variable instead.